Server/Test_Plan_Templateeditor.md

```diff
@@ -0,0 +1,22 @@
+## Templateeditor Test Plan
+
+
+
+| Test Case                                | Expected Result                          | Result         | Related Comment |
+| ---------------------------------------- | ---------------------------------------- | -------------- | --------------- |
+| Enable templateeditor.                   | App is enabled and can be used in the admin settings. | 🚧             |                 |
+| Have two themes themeA and themeB in the server installed.<br />Enable themeA, go to the admin settings load email templates. Disable themeA,  enable themeB, go to the admin settings again load again email templates. | Templateeditor can be used with both themes. | 🚧             |                 |
+| **Enable an email server to actually send  emails, for example mailhog** |                                          |                |                 |
+| <br /> In the admin settings enable this value:<br />Allow users to share via link - Allow users to send mail notification for shared files<br />Choose template 'Sharing email - public link shares (HTML)', edit the default content of the template.<br /> Create a regular user 'user1', as user1 share by link a file, add an email notification. | An email is sent following the edited template. | 🚧             |                 |
+| <br /> In the admin settings enable this value:<br />Allow users to share via link - Allow users to send mail notification for shared files<br />Choose template 'Sharing email - public link shares (plain text fallback)', edit the default content of the template.<br /> Create a regular user 'user1', as user1 share by link a file, add an email notification.<br /> | An email is sent following the edited template. | 🚧             |                 |
```

Comment:
Maybe add that we must check that the file name and the username of the user are shown in the received mail.